EventsExpress/Controllers/NotificationTemplateController.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
+using AutoMapper;
+using EventsExpress.Core.DTOs;
+using EventsExpress.Core.IServices;
+using EventsExpress.Db.Enums;
+using EventsExpress.ViewModels;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+
+namespace EventsExpress.Controllers
+{
+    [Authorize(Roles = "Admin")]
+    [Route("api/[controller]")]
+    [ApiController]
+    public class NotificationTemplateController : Controller
+    {
+        private readonly INotificationTemplateService _notificationTemplateService;
+        private readonly IMapper _mapper;
+
+        public NotificationTemplateController(INotificationTemplateService notificationTemplateService, IMapper mapper)
+        {
+            _notificationTemplateService = notificationTemplateService;
+            _mapper = mapper;
+        }
+
+        [HttpGet("All")]
+        public async Task<ActionResult<IEnumerable<NotificationTemplateDTO>>> GetAll([FromQuery] int page, [FromQuery] int pageSize)
```

Comment:
Just noticed, that you have paging set up in controller, but there is no page control on UI. Let's remove the paging functionality from controller and router altogether.